tests/test_dcm/test_header_tweaks.py

```diff
@@ -0,0 +1,208 @@
+# Copyright (C) 2019 Simon Biggs
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version (the "AGPL-3.0+").
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License and the additional terms for more
+# details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# ADDITIONAL TERMS are also included as allowed by Section 7 of the GNU
+# Affero General Public License. These additional terms are Sections 1, 5,
+# 6, 7, 8, and 9 from the Apache License, Version 2.0 (the "Apache-2.0")
+# where all references to the definition "License" are instead defined to
+# mean the AGPL-3.0+.
+
+# You should have received a copy of the Apache-2.0 along with this
+# program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
+
+
+import os
+import subprocess
+
+import numpy as np
+
+import pydicom
+
+from pymedphys.dcm import (
+    dcm_from_dict, adjust_machine_name, adjust_rel_elec_density)
+
+
+HERE = os.path.dirname(__file__)
+ORIGINAL_DICOM_FILENAME = os.path.join(HERE, 'original.dcm')
+ADJUSTED_DICOM_FILENAME = os.path.join(HERE, 'adjusted.dcm')
+
+
+def test_adjust_machine_name():
+    new_name = 'new_name'
+
+    original_dicom_file = dcm_from_dict({
+        'BeamSequence': [
+            {
+                'TreatmentMachineName': 'hello'
+            },
+            {
+                'TreatmentMachineName': 'george'
+            }
+        ]
+    })
+
+    expected_dicom_file = dcm_from_dict({
+        'BeamSequence': [
+            {
+                'TreatmentMachineName': new_name
+            },
+            {
+                'TreatmentMachineName': new_name
+            }
+        ]
+    })
+
+    adjusted_dicom_file = adjust_machine_name(original_dicom_file, new_name)
+
+    assert adjusted_dicom_file != original_dicom_file
+    assert adjusted_dicom_file == expected_dicom_file
+
+    pydicom.write_file(ORIGINAL_DICOM_FILENAME, original_dicom_file)
+    subprocess.check_call(
+        'pymedphys dicom adjust-machine-name'.split() +
+        [ORIGINAL_DICOM_FILENAME, ADJUSTED_DICOM_FILENAME, new_name]
+    )
```

Comment:
Testing the CLI as well as the python interface.

Reply:
I think a broader "dicom edit" tool for any tag might be more useful here. Not urgent or necessary now - just a thought :)